Integrations/integration-AlienVaultOTX.yml

```diff
@@ -452,3 +459,6 @@ script:
     description: Query url in AlienVault OTX
   runonce: false
 fromversion: 3.0.1
+releaseNotes: fixing !url plus adding a test
```

Comment:
please change to "Fix !url to try extract base url, and return a readable error in case of failure"